commit 919c0d14ae93073a3957c018a6d86ceb1e2e454b
Merge: de08341a0ef7 6b1ed9086592
Author: Linus Torvalds <torvalds@linux-foundation.org>
Date:   Tue Jun 24 18:09:06 2008 -0700

    Merge branch 'kvm-updates-2.6.26' of git://git.kernel.org/pub/scm/linux/kernel/git/avi/kvm
    
    * 'kvm-updates-2.6.26' of git://git.kernel.org/pub/scm/linux/kernel/git/avi/kvm:
      KVM: Remove now unused structs from kvm_para.h
      x86: KVM guest: Use the paravirt clocksource structs and functions
      KVM: Make kvm host use the paravirt clocksource structs
      x86: Make xen use the paravirt clocksource structs and functions
      x86: Add structs and functions for paravirt clocksource
      KVM: VMX: Fix host msr corruption with preemption enabled
      KVM: ioapic: fix lost interrupt when changing a device's irq
      KVM: MMU: Fix oops on guest userspace access to guest pagetable
      KVM: MMU: large page update_pte issue with non-PAE 32-bit guests (resend)
      KVM: MMU: Fix rmap_write_protect() hugepage iteration bug
      KVM: close timer injection race window in __vcpu_run
      KVM: Fix race between timer migration and vcpu migration

diff --cc arch/x86/xen/Kconfig
index 525b108411bd,3a4f16aea4bf..6c388e593bc8
--- a/arch/x86/xen/Kconfig
+++ b/arch/x86/xen/Kconfig
@@@ -5,8 -5,9 +5,9 @@@
  config XEN
  	bool "Xen guest support"
  	select PARAVIRT
+ 	select PARAVIRT_CLOCK
  	depends on X86_32
 -	depends on X86_CMPXCHG && X86_TSC && !(X86_VISWS || X86_VOYAGER)
 +	depends on X86_CMPXCHG && X86_TSC && X86_PAE && !(X86_VISWS || X86_VOYAGER)
  	help
  	  This is the Linux Xen port.  Enabling this will allow the
  	  kernel to boot in a paravirtualized environment under the